### ISSUE TITLE
Images and themes not being loaded in Bolt 4.1.3

### ISSUE THREAD

**Issue by prbt2016** [bolt/core#2050]
Hello,

I  was in the process of manual installation of bolt 4.1.3 in subdirectory i.e /home/user/public_html/bolt413 on PHP  7.2.24 , MYSQL 5.5.62 and Apache 2.2.34.

I followed the steps mentioned here in the installation manual on official site https://docs.bolt.cm/4.0/installation/installation

Here are the steps done :

1. Download bolt package via composer

php composer.phar create-project bolt/project bolt413

2. Set up database details in .env file(I use MYSQL)
`DATABASE_URL=mysql://db_user:"db_password"@localhost:3306/db_name `

3. Entered this command :

     `bin/console bolt:setup `

and entered all the  required admin details like Username , Password , Email and Display Name

I get : [OK] Bolt was set up successfully! Start a web server, and open your Bolt site in a browser. However images and css are not loaded properly after hitting http://{{DOMAIN.COM}}/{{BOLT-DIR}}/public :

![bolt](https://user-images.githubusercontent.com/17871448/97082659-f59ad500-1628-11eb-9ec0-fb65b31bd2a9.png)

4. Also I checked in network panel in developer tools in browser that the image and theme path is being searched in 

http://{{DOMAIN.COM}}/theme and http://{{DOMAIN.COM}}/thumbs instead of 
http://{{DOMAIN.COM}}/{{BOLT-DIR}}/thumbs

How to fix this issue ?.  

Also how to install Bolt by keeping only public directory contents inside webroot and other files and folders outside of webroot?

As on the official site https://docs.bolt.cm/4.0/howto/troubleshooting-outside-webroot link is broken.

Is there any specific path configuration that needs to be done for the paths?

**Comment by simongroenewolt** [comment]
I think you might have overlooked the note in this section https://docs.bolt.cm/4.0/installation/installation#starting-a-webserver

**Comment by jameswill** [comment]
Hi,

I am also getting the same issue.

Can we install it in a subdirectory ?

Previously in Bolt 3.7.x mapping was allowed so that one can keep the public directory data into web accessible directory and put all other contents in non-web accessible directory. Can this happen in Bolt 4.1.4 as well ?

**Comment by simongroenewolt** [comment]
This is what the note says:

> Note: The folder you've just created has a public/ folder. This is the actual web root of the site. If you're not using one of the options below, but are configuring a webserver yourself, make sure you use public/ as the web root. Bolt does not support putting all of its files inside the webroot itself, as that is considered to be a bad practice.

So configure your webserver to use `{{BOLT-DIR}}/public` as the web root.

**Comment by bobdenotter** [comment]
> Can we install it in a subdirectory ?

Nope, that's not possible, because it would mean putting all sensitive files in a place where they'd potentially be accessible.

> Previously in Bolt 3.7.x mapping was allowed so that one can keep the public directory data into web accessible directory and put all other contents in non-web accessible directory. Can this happen in Bolt 4.1.4 as well ?

Yes, you can put the entire bolt installation in one spot, say `~/website` and then in make a symlink for example from `/var/www/html` -> `~/website/public`.
